src/ui/components/Rating/index.js

```diff
@@ -68,6 +68,7 @@ export class RatingBase extends React.Component {
         id: `Rating-rating-${thisRating}`,
         key: `rating-${thisRating}`,
         ref: (ref) => { this.ratingElements[thisRating] = ref; },
+        title: !readOnly ? `Rate this add-on ${thisRating} out of 5.` : null,
```

Comment:
This needs to be translated using `i18n.gettext` and `i18n.sprintf`. You can find examples on how to use them elsewhere in the app.